lobby.properties

```diff
@@ -0,0 +1,5 @@
+## uncomment the following lines to have local game runners connect
```

Comment:
This file will be read by the code. I do not know if the code is smart enough to default to valid values when it does not find them here. It certainly defaults to other values when this file is not found. Please confirm the behavior.